app/helpers/homepage_helper.rb

```diff
@@ -0,0 +1,23 @@
+module HomepageHelper
+  def organisations_path
+    Plek.find("content-store") + "/content/government/organisations"
```

Comment:
Anyone know, is this the correct way to be using Plek?

I'm not sure if I need a method like:
```
  def base_path
    Rails.env.production? ? Plek.new.website_root : Plek.find("content-store")
  end
```

Plek seems to change based on an environment variable, do we usually have those set to `/` or `https://gov.uk/` in prod?

Reply:
It looks like there's a `GdsApi.organisations` you can use for this